src/app/forestillinger/[slug]/page.tsx

```diff
@@ -7,9 +7,67 @@ import PlayBanner from '@/components/Plays/PlayBanner';
 import ImageGallery from '@/components/ImageGallery';
 import ContributorsSection from '@/components/Plays/ContributorsSection';
 import { notFound } from 'next/navigation';
+import { Metadata } from 'next';
 
-const Page = async ({ params }) => {
-    const { slug } = (await params) as { slug: string };
+type Props = {
+    params: Promise<{ slug: string }>;
+};
+
+export async function generateMetadata({ params }: Props): Promise<Metadata> {
+    const slug = (await params).slug;
+
+    const play: Play = await getPlay(slug);
+    const landingPage = await getMainBanner();
+
+    const highlightedPlay = landingPage.highlightedPlays.find(
+        (highlightedPlay) => highlightedPlay.title === play.playTitle
+    );
```

Comment:
Her er det litt synd at på hver forestilling så må vi gjøre to fetches, en for forestillingsdata og en for å få kort beskrivelsen fra highlightedPlays. Det er ikke et problem da, men ville bare påpeke at jeg vet det ikke er ideelt 😅